change/@fluentui-react-experiments-a9e50579-3134-4efe-9761-a2764262647f.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "Removing Button, MenuButton, SplitButton and Toggle components. Deprecating Slider component. Updating MicroFeedback component to use Button in @fluentui/react instead of the removed Button in @fluentui/react-experiments.",
```

Comment:
```suggestion
  "comment": "[BREAKING] Removing Button, MenuButton, SplitButton and Toggle components. Deprecating Slider component. Updating MicroFeedback component to use Button in @fluentui/react instead of the removed Button in @fluentui/react-experiments.",
```